xarray/tests/test_backends.py

```diff
@@ -1278,10 +1289,11 @@ class TestNetCDF4ViaDaskData(TestNetCDF4Data):
     @contextlib.contextmanager
     def roundtrip(self, data, save_kwargs={}, open_kwargs={},
                   allow_cleanup_failure=False):
+        open_kwargs['chunks'] = open_kwargs.get('chunks', 1)
```

Comment:
There's a mutating mutable default values issue here -- this is changing `open_kwargs` for every invocation.

Instead, we should make `open_kwargs=None` and then set the chunks keyword argument as is approrpriate.

Also: use `open_kwargs.setdefault('chunks', -1)` instead. `setdefault()` is a little more succinct, and `-1` means "full sized chunks" which will be faster for our test suite.

Reply:
@shoyer - good points. Addressed in my latest commit.